### PR TITLE
Change the parameter name from `classifier` to `regressor` in the code snippet of `README.md`

### PR DESCRIPTION
**Pull request by g-votte**
This is definitely not a classifier but a regressor.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ We use the terms *study* and *trial* as follows:
 - Trial: a single execution of the objective function
 
 Please refer to sample code below. The goal of a *study* is to find out the optimal set of
-hyperparameter values (e.g., `classifier` and `svm_c`) through multiple *trials* (e.g.,
+hyperparameter values (e.g., `regressor` and `svm_c`) through multiple *trials* (e.g.,
```
Comment (toshihikoyanase) [inline README.md:58]: How about updating the hyperparameter name of `SVR` as well?
```suggestion
hyperparameter values (e.g., `regressor` and `svr_c`) through multiple *trials* (e.g.,
```

Comment (g-votte) [reply README.md]: Ah, good catch!

**Review by toshihikoyanase — APPROVED** [review]
LGTM! Thank you.

**Comment by toshihikoyanase** [comment]
Let me merge this PR with a single approval since it only updates README, not code.

**Pull request merged**